### PR TITLE
🚨🚨 Source MetaBase: Remove `Activity` Stream

### PR DESCRIPTION
**Pull request by artem1205**
## What

Fix Source MetaBase Connector health report

## How
Remove `Activity` Stream
Reason: [Metabase API release v0.46.4](https://github.com/metabase/metabase/releases/tag/v0.46.4)

## Recommended reading order
1. `manifest.yaml`

## 🚨 User Impact 🚨

Stream removed, refresh source schema is needed. 

Playbook: [Source MetaBase Breaking Change Release Playbook](https://docs.google.com/document/d/13rNiYEQtJ-6h-pTTQoDshJsPO7kBYEFr0AHWlS40goI/edit)

## Pre-merge Actions

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

### CODE REVIEW

**Comment by octavia-squidington-iii** [comment]
##  source-metabase test report (commit [`2ceb105eab`](https://github.com/airbytehq/airbyte/pull/27777/commits/2ceb105eaba707043a66ad1c6b510f592a8a40a8)) - ❌

⏲️  Total pipeline duration: 139 seconds

| Step                                                                   | Result   |
|:-----------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-metabase/metadata.yaml | ✅        |
| Connector version semver check                                         | ✅        |
| Connector version increment check                                      | ✅        |
| QA checks                                                              | ✅        |
| Code format checks                                                     | ✅        |
| Connector package install                                              | ✅        |
| Build source-metabase docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                             | ✅        |
| Acceptance tests                                                       | ❌        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/artem1205_con-health-fix-metabase/1687897359/2ceb105eaba707043a66ad1c6b510f592a8a40a8/source-metabase/0.4.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-metabase test
```

**Comment by octavia-squidington-iii** [comment]
##  source-metabase test report (commit [`2ceb105eab`](https://github.com/airbytehq/airbyte/pull/27777/commits/2ceb105eaba707043a66ad1c6b510f592a8a40a8)) - ❌

⏲️  Total pipeline duration: 130 seconds

| Step                                                                   | Result   |
|:-----------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-metabase/metadata.yaml | ✅        |
| Connector version semver check                                         | ✅        |
| Connector version increment check                                      | ✅        |
| QA checks                                                              | ✅        |
| Code format checks                                                     | ✅        |
| Connector package install                                              | ✅        |
| Build source-metabase docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                             | ✅        |
| Acceptance tests                                                       | ❌        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/artem1205_con-health-fix-metabase/1687897598/2ceb105eaba707043a66ad1c6b510f592a8a40a8/source-metabase/0.4.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-metabase test
```

**Comment by octavia-squidington-iii** [comment]
##  source-metabase test report (commit [`7281375fb3`](https://github.com/airbytehq/airbyte/pull/27777/commits/7281375fb3154009da17a9b782d8f057ef854089)) - ❌

⏲️  Total pipeline duration: 122 seconds

| Step                                                                   | Result   |
|:-----------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-metabase/metadata.yaml | ✅        |
| Connector version semver check                                         | ✅        |
| Connector version increment check                                      | ✅        |
| QA checks                                                              | ❌        |
| Code format checks                                                     | ✅        |
| Connector package install                                              | ✅        |
| Build source-metabase docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                             | ✅        |
| Acceptance tests                                                       | ✅        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/artem1205_con-health-fix-metabase/1687899369/7281375fb3154009da17a9b782d8f057ef854089/source-metabase/1.0.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-metabase test
```

**Comment by octavia-squidington-iii** [comment]
##  source-metabase test report (commit [`7281375fb3`](https://github.com/airbytehq/airbyte/pull/27777/commits/7281375fb3154009da17a9b782d8f057ef854089)) - ❌

⏲️  Total pipeline duration: 125 seconds

| Step                                                                   | Result   |
|:-----------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-metabase/metadata.yaml | ✅        |
| Connector version semver check                                         | ✅        |
| Connector version increment check                                      | ✅        |
| QA checks                                                              | ❌        |
| Code format checks                                                     | ✅        |
| Connector package install                                              | ✅        |
| Build source-metabase docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                             | ✅        |
| Acceptance tests                                                       | ✅        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/artem1205_con-health-fix-metabase/1687899604/7281375fb3154009da17a9b782d8f057ef854089/source-metabase/1.0.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-metabase test
```

**Comment by octavia-squidington-iii** [comment]
##  source-metabase test report (commit [`c3debdf77b`](https://github.com/airbytehq/airbyte/pull/27777/commits/c3debdf77be21739e4b9bb82cb40d45f0747f420)) - ✅

⏲️  Total pipeline duration: 147 seconds

| Step                                                                   | Result   |
|:-----------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-metabase/metadata.yaml | ✅        |
| Connector version semver check                                         | ✅        |
| Connector version increment check                                      | ✅        |
| QA checks                                                              | ✅        |
| Code format checks                                                     | ✅        |
| Connector package install                                              | ✅        |
| Build source-metabase docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                             | ✅        |
| Acceptance tests                                                       | ✅        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/artem1205_con-health-fix-metabase/1687901754/c3debdf77be21739e4b9bb82cb40d45f0747f420/source-metabase/1.0.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-metabase test
```

**Comment by octavia-squidington-iii** [comment]
##  source-metabase test report (commit [`d0fa762871`](https://github.com/airbytehq/airbyte/pull/27777/commits/d0fa762871cc3f2751bdf7c5035c6c0f110db4b2)) - ✅

⏲️  Total pipeline duration: 114 seconds

| Step                                                                   | Result   |
|:-----------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-metabase/metadata.yaml | ✅        |
| Connector version semver check                                         | ✅        |
| Connector version increment check                                      | ✅        |
| QA checks                                                              | ✅        |
| Code format checks                                                     | ✅        |
| Connector package install                                              | ✅        |
| Build source-metabase docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                             | ✅        |
| Acceptance tests                                                       | ✅        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/artem1205_con-health-fix-metabase/1687902105/d0fa762871cc3f2751bdf7c5035c6c0f110db4b2/source-metabase/1.0.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-metabase test
```

**Review by davydov-d — APPROVED** [review]

File: airbyte-integrations/connectors/source-metabase/integration_tests/configured_catalog.json [hunk 1]
```diff
@@ -1,15 +1,5 @@
 {
```
[the remark this answers is not in the document]
Comment (artem1205) [reply airbyte-integrations/connectors/source-metabase/integration_tests/configured_catalog.json:1]: done

File: airbyte-integrations/connectors/source-metabase/acceptance-test-config.yml [hunk 2]
```diff
@@ -2,37 +2,39 @@ connector_image: airbyte/source-metabase:dev
...
-        bypass_reason: "data changes very fast"
-      - name: dashboards
-        bypass_reason: "data changes very fast"
-      - name: users
-        bypass_reason: "data changes very fast"
-      fail_on_extra_columns: false
+      - config_path: "secrets/config.json"
+        configured_catalog_path: "integration_tests/configured_catalog.json"
+        empty_streams:
+          - name: activity
```
[the remark this answers is not in the document]
Comment (artem1205) [reply airbyte-integrations/connectors/source-metabase/acceptance-test-config.yml]: done

**Comment by octavia-squidington-iii** [comment]
##  source-metabase test report (commit [`5ca7078848`](https://github.com/airbytehq/airbyte/pull/27777/commits/5ca70788480f5cf4bafb1a3a25f46bf3d86a1d41)) - ✅

⏲️  Total pipeline duration: 131 seconds

| Step                                                                   | Result   |
|:-----------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-metabase/metadata.yaml | ✅        |
| Connector version semver check                                         | ✅        |
| Connector version increment check                                      | ✅        |
| QA checks                                                              | ✅        |
| Code format checks                                                     | ✅        |
| Connector package install                                              | ✅        |
| Build source-metabase docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                             | ✅        |
| Acceptance tests                                                       | ✅        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/artem1205_con-health-fix-metabase/1688029151/5ca70788480f5cf4bafb1a3a25f46bf3d86a1d41/source-metabase/1.0.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-metabase test
```

**Review by lazebnyi — APPROVED** [review]

**Pull request merged**